bin/manala

```diff
@@ -15,6 +15,8 @@ require_once __DIR__.'/../vendor/autoload.php';
 use Manala\Command;
 use Symfony\Component\Console\Application;
 
+define('MANALA_DIR', \Phar::running() ?: __DIR__.'/../');
```

Comment:
Out of curiosity: is the `\Phar::running` call needed ? Won't the path always be `__DIR__.'/../'` whether you are running the phar or the php script ?